src/main/resources/hudson/plugins/gradle/injection/GradleEnterpriseErrorsAction/index.jelly

```diff
@@ -0,0 +1,51 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:i="jelly:fmt">
+    <l:layout title="Gradle Enterprise" permission="${app.ADMINISTER}">
+        <l:main-panel>
+            <div class="jenkins-app-bar">
+                <div class="jenkins-app-bar__content">
+                    <h1>Gradle Enterprise</h1>
+                </div>
+            </div>
+            <section class="jenkins-section">
+                <h2 class="jenkins-section__header">${%Unexpected errors}</h2>
+                <p>Errors that were not gracefully handled by the Gradle Enterprise plugin</p>
+                <table id="gradle-enterprise-errors" class="jenkins-table sortable">
+                    <thead>
+                        <tr>
+                            <th class="medium-column">${%Project}</th>
+                            <th class="medium-column" style="text-align: center">${%Build Tool}</th>
+                            <th class="small-column" style="text-align: center">${%Outcome}</th>
+                            <th class="small-column" initialSortDir="up">${%Start time}</th>
+                            <th class="medium-column">${%URL}</th>
+                        </tr>
+                    </thead>
+                    <j:forEach items="${it.errors}" var="e">
+                        <tr>
+                            <td class="medium-column">${e.project}</td>
+                            <td class="small-column center">
+                                <j:choose>
+                                    <j:when test="${e.buildTool == 'GRADLE'}">
```

Comment:
Since all the logic is on the server side, maybe we can just return an icon from the server side as well? Then we don't need any conditions here and all the related logic will be in one place. It will be easier to add new tools in the future.

Reply:
yep, thought about this as well, will refactor towards this